components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/listener/ApplicationResourceManagementListener.java

```diff
@@ -63,7 +63,7 @@ default int getExecutionOrderId() {
      *
      * @return true if enabled
      */
-    default boolean isEnable() {
+    default boolean isEnabled() {
```

Comment:
If this was an existing API, better not to change it.

Reply:
This is a new listener introduced for resourceId based application management.